guides/common/modules/proc_recovering-corrupted-content-on-a-smart-proxy.adoc

```diff
@@ -0,0 +1,34 @@
+[id="Recovering_Corrupted_Content_on-a-Smart-Proxy_{context}"]
+= Recovering corrupted content on a {SmartProxy}
+
+In case of content corruption on a {SmartProxy}, you can recover it by using the `verify-checksum` command in Hammer CLI.
+The `verify-checksum` command works for optional parameters associated with the {SmartProxy} such as `content-view id`, `environment-id`, or `repository-id` to limit the repair.
+You can run the `verify-checksum` command without any filters so it runs on all content on the {SmartProxy}.
+
+.CLI procedure
+* To run a `verify-checksum` command in Hammer CLI for `content-view id`:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+hammer capsule content verify-checksum --id 2 --organization-id 1 --content-view-id 3
+----
+* To run a `verify-checksum` command in Hammer CLI for `environment-id`:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+hammer capsule content verify-checksum --id 2 --organization-id 1 --lifecycle-environment-id 1
+----
+* To run a `verify-checksum` command in Hammer CLI for `repository-id`:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+hammer capsule content verify-checksum --id 2 --organization-id 1 --repository-id 1
+----
+After running a `verify-checksum` command, it will appear on the *Tasks* page as the action *Verify checksum for content on smart proxy*.
```

Comment:
This line applies to all the commands with or without parameters, doesn't it? I'd move it up after the intro.